student/src/main/java/tds/student/sql/repository/remote/AssessmentRepository.java

```diff
@@ -1,4 +1,4 @@
-package tds.student.sql.repository;
+package tds.student.sql.repository.remote;
```

Comment:
Next time you want to do something like this maybe do it separately from the feature request.